### PR TITLE
feat: add dialog title and styles

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

File: src/components/Menu.js [hunk 1]
```diff
@@ -10,17 +10,23 @@ const refresh = new URL('../assets/icons/refresh.svg', import.meta.url).href;
...
 const renderMenu = menuItem => {
+  const output = {};
   switch (menuItem) {
     case 'info':
-      return html`
+      output.title = 'info';
+      output.template = html`
         info
       `;
+      break;
```
Comment (LarsDenBakker) [inline src/components/Menu.js]: Why not `return { title: 'info', template: html`...`  }` For all of these?

Comment (thepassle) [reply src/components/Menu.js]: much better 👍

**Comment by LarsDenBakker** [comment]
@thepassle LGTM, but there is a conflict :)

**Comment by thepassle** [comment]
@LarsDenBakker fixed

**Pull request merged**